compute/apiv1/smoke_test.go

```diff
@@ -396,3 +452,125 @@ func TestPaginationMapResponseMaxRes(t *testing.T) {
 		t.Error("Couldn't find the accelerator in the response")
 	}
 }
+
+func TestTypeInt64(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping smoke test in short mode")
+	}
+	ctx := context.Background()
+	c, err := NewImagesRESTClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	space := uid.NewSpace("gogapic", nil)
+	name := space.New()
+
+	codes := []int64{
+		5543610867827062957,
+	}
+	req := &computepb.InsertImageRequest{
+		Project: projectId,
+		ImageResource: &computepb.Image{
+			Name:         &name,
+			LicenseCodes: codes,
+			SourceImage:  proto.String("projects/debian-cloud/global/images/debian-10-buster-v20210721"),
+		},
+	}
+	globalClient, err := NewGlobalOperationsRESTClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	insert, err := c.Insert(ctx, req)
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = globalClient.Wait(ctx,
+		&computepb.WaitGlobalOperationRequest{
+			Project:   projectId,
+			Operation: insert.Proto().GetName(),
+		})
+	if err != nil {
+		t.Error(err)
+	}
+
+	defer func(c *ImagesClient, ctx context.Context, req *computepb.DeleteImageRequest) {
+		_, err := c.Delete(ctx, req)
+		if err != nil {
+		}
+	}(c, ctx, &computepb.DeleteImageRequest{
+		Project: projectId,
+		Image:   name,
+	})
+
+	fetched, err := c.Get(ctx,
+		&computepb.GetImageRequest{
+			Project: projectId,
+			Image:   name,
+		})
+	if !reflect.DeepEqual(fetched.GetLicenseCodes(), codes) {
+		t.Fatal("License codes are not equal")
+	}
+}
+
+func TestCapitalLetter(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping smoke test in short mode")
+	}
+	ctx := context.Background()
+	c, err := NewFirewallsRESTClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	space := uid.NewSpace("gogapic", nil)
+	name := space.New()
+	allowed := []*computepb.Allowed{
+		{
+			IPProtocol: proto.String("tcp"),
+			Ports: []string{
+				"80",
+			},
+		},
+	}
+	res := &computepb.Firewall{
+		SourceRanges: []string{
+			"0.0.0.0/0",
+		},
+		Name:    proto.String(name),
+		Allowed: allowed,
+	}
+	req := &computepb.InsertFirewallRequest{
+		Project:          projectId,
+		FirewallResource: res,
+	}
+	insert, err := c.Insert(ctx, req)
+	if err != nil {
+		t.Fatal(err)
+	}
+	globalClient, err := NewGlobalOperationsRESTClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = globalClient.Wait(ctx,
+		&computepb.WaitGlobalOperationRequest{
+			Project:   projectId,
+			Operation: insert.Proto().GetName(),
+		})
+	if err != nil {
+		t.Error(err)
+	}
+	defer func(c *FirewallsClient, ctx context.Context, req *computepb.DeleteFirewallRequest) {
+		_, err := c.Delete(ctx, req)
+		if err != nil {
+		}
```

Comment:
Is this necessary if there it is never handled?

Reply:
now handled